tests/test_projectbuilder.py

```diff
@@ -233,6 +233,17 @@ def test_missing_outdir(mocker, tmp_dir, test_flit_path):
     assert os.path.isdir(out)
 
 
+def test_relative_outdir(mocker, tmp_dir, test_flit_path):
+    mocker.patch('importlib.import_module', autospec=True)
+    mocker.patch('pep517.wrappers.Pep517HookCaller', autospec=True)
+
+    builder = build.ProjectBuilder(test_flit_path)
+
+    builder.build('sdist', '.')
+
+    builder.hook.build_sdist.assert_called_with(os.path.realpath('.'), {})
```

Comment:
You know what the path is already, right? Can you instead do:

`assert_called_with(tmp_dir, {})` or whatever `.` is intended to resolve to?

That would help to detect bugs from `os.realpath` doing something different than you expect.

Reply:
Sure.